packages/hub/src/components/EntityInfo.tsx

```diff
@@ -1,29 +1,76 @@
-import { FC } from "react";
+import { FC, cloneElement, ReactNode } from "react";
 import Link from "next/link";
 
-import { userRoute } from "@/routes";
+import { clsx } from "clsx";
+import { IconProps } from "@/relative-values/components/ui/icons/Icon";
 
 // works both for models and for definitions
-export const EntityInfo: FC<{
-  username: string;
+export type entityNode = {
   slug: string;
   href: string;
-}> = ({ username, slug, href }) => {
+  icon?: FC<IconProps>;
+};
+
+const ListWithSeparator: React.FC<{
+  items: ReactNode[];
+  separator: (r: number) => ReactNode;
+}> = ({ items, separator }) => {
+  return items.reduce<React.ReactNode[]>((accumulator, item, index) => {
+    const isLastItem = index === items.length - 1;
+    return isLastItem
+      ? [...accumulator, item]
+      : [...accumulator, item, separator(index)];
+  }, []);
+};
+
+const Entity: FC<entityNode & { isFirst: boolean; isLast: boolean }> = ({
+  slug,
+  href,
+  icon: Icon,
+  isFirst,
+  isLast,
+}) => {
+  return (
+    <Link
+      className={clsx(
+        "text-lg text-slate-700 hover:underline flex items-center gap-1 group pt-1 py-2 pr-3",
+        isLast ? "font-semibold" : "",
+        !isFirst && "pl-3"
+      )}
+      href={href}
+      key={href}
+    >
+      {Icon && (
+        <Icon
+          className="text-slate-600 opacity-50 group-hover:opacity-100 transition mr-1"
+          size={18}
+        />
+      )}
+      {slug}
+    </Link>
+  );
+};
+
+export const EntityInfo: FC<{
+  nodes: entityNode[];
+}> = ({ nodes }) => {
+  const links = nodes.map((node, i) => (
+    <Entity
+      {...node}
+      key={i}
+      isFirst={i === 0}
+      isLast={i === nodes.length - 1}
+    />
+  ));
+  const separator = (key: number) => (
+    <div key={`s${key}`} className="text-lg text-gray-300">
```

Comment:
Separator is ~1px lower than it should be:
<img width="1179" alt="Captura de pantalla 2023-07-10 a la(s) 15 03 11" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/c303ee01-f6e6-4761-9e0b-bcb3d9f371a7">

Reply:
I think that's because of `pt-1 py-2` in `<Entity>`; that padding could be lifted to `EntityInfo`.

Reply:
Oh, good point.